code-challenges.test.js

```diff
@@ -24,25 +29,90 @@
 
 // b) Create the function that makes the test pass.
 
+//create fib variable
+//for fib define num values 
+//for loop states that the function will act on numbers greater than 2, but less than or equal to the example inputs (6,10). Function will iterate through the given parameters
+//return array of 6 and 10 in fib.
+
+
+
+
+var i;
+var fib = [];
+
+fib[0] = 0;
+fib[1] = 1;
+for (i = 2; i <= 10; i++) {
+  fib[i] = fib[i - 2] + fib[i - 1];
+  console.log(fib[i]);
+}
+--------------------------------------
+var i;
+var fib = [];
+
+fib[0] = 0;
+fib[1] = 1;
+for (i = 2; i <= 6; i++) {
+  fib[i] = fib[i - 2] + fib[i - 1];
+  console.log(fib[i]);
+}
 
 // --------------------2) Create a function that takes in an array and returns a new array of only odd numbers sorted from least to greatest.
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const fullArr1 = [4, 9, 0, "7", 8, true, "hey", 7, 199, -9, false, "hola"]
+// const fullArr1 = [4, 9, 0, "7", 8, true, "hey", 7, 199, -9, false, "hola"]
 // Expected output: [-9, 7, 9, 199]
 
-const fullArr2 = ["hello", 7, 23, -823, false, 78, null, "67", 6, "number"]
-// Expected output: [-823, 7, 23]
+// const fullArr2 = ["hello", 7, 23, -823, false, 78, null, "67", 6, "number"]
+// // Expected output: [-823, 7, 23]
+
+describe("filteredNumbers", () => {
+    it("takes in an array and returns a new array of only odd numbers sorted from least to greatest.",
+  () => }
+  expect(filteredNumbers(fullArr1, fullArr2)).toEqual(filteredNumbers)
```

Comment:
For this question, you will consider each variable in separate expect statements with the respective output appended to the end
For example, 
expect(filteredNumbers(fullArr1)).toEqual([-9, 7, 9, 199])